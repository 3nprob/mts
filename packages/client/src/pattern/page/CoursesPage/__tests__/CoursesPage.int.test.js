import { async, regular } from './CoursesPage.int.story'

jest.mock('react-router-dom', () => ({ Link: () => <div /> }))

describe('<CoursesPage />', () => {
  afterEach(() => jest.clearAllMocks())

  describe('Render', () => {
    test('must match async', () => {
      expect(global.renderToJSON(async())).toMatchSnapshot()
    })

    test('must match regular', () => {
      expect(global.renderToJSON(regular())).toMatchSnapshot()
    })
  })
})
