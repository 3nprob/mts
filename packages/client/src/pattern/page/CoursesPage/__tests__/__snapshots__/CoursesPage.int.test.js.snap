// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CoursesPage /> Snaphot must match asyncRender 1`] = `null`;

exports[`<CoursesPage /> Snaphot must match defaultRender 1`] = `
<div
  className=""
>
  <div
    className="coursesTemplate"
  >
    <nav
      className="nav"
      itemScope=""
      itemType="http://schema.org/SiteNavigationElement"
      role="navigation"
    >
      <div
        className="navBar"
      >
        <div>
          some link
        </div>
        <ul
          className="navBarList"
        >
          <li
            className="navBarListItem"
          >
            <div>
              some link
            </div>
          </li>
          <li
            className="navBarListItem"
          >
            <div>
              some link
            </div>
          </li>
          <li
            className="navBarListItem"
          >
            <div>
              some link
            </div>
          </li>
        </ul>
        <div
          className="navBarMobileToggle"
        >
          <button
            aria-label="Toggle Mobile Navigation Open/Close"
            className="navBarMobileToggleButton"
            onClick={[Function]}
            type="button"
          >
            <svg
              height="24px"
              viewBox="0 0 124 124"
              width="24px"
            >
              <path
                d="M112 6H12C5.4 6 0 11.4 0 18s5.4 12 12 12h100c6.6 0 12-5.4 12-12s-5.4-12-12-12zM112 50H12C5.4 50 0 55.4 0 62s5.4 12 12 12h100c6.6 0 12-5.4 12-12s-5.4-12-12-12zM112 94H12c-6.6 0-12 5.4-12 12s5.4 12 12 12h100c6.6 0 12-5.4 12-12s-5.4-12-12-12z"
              />
            </svg>
          </button>
        </div>
      </div>
    </nav>
    <div
      className=""
      style={
        Object {
          "display": "flex",
          "justifyContent": "center",
          "marginTop": "55px",
        }
      }
    >
      <div
        className="container"
      >
        <div
          className="containerLogo"
        >
          <div
            className="containerLogoImage"
          />
        </div>
        <div
          className="containerSessionLinks"
        >
          <div
            className="containerSessionLinksMenu"
          >
            <div>
              <span
                className="containerSessionLinksMenuItem"
                role="menuitem"
                tabIndex="0"
              >
                <div>
                  <div
                    className="containerSessionLinksMenuItemText"
                  >
                    <div>
                      My Courses
                    </div>
                  </div>
                </div>
              </span>
            </div>
            <div
              className="containerSessionLinksMenuSelect"
            >
              <button
                className="containerSessionLinksMenuSelectButton"
                tabIndex="0"
                type="button"
              >
                <div>
                  <svg
                    className="containerSessionLinksMenuSelectButtonImage"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M12 8c1.1 0 2-.9 2-2s-.9-2-2-2-2 .9-2 2 .9 2 2 2zm0 2c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm0 6c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2z"
                    />
                  </svg>
                </div>
              </button>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
